### PR TITLE
Use maven-metadata shields.io badge for gradle

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ![beta](https://img.shields.io/badge/stability-beta-darkorange.svg)
 [![Maven Central](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/google/cloud/tools/jib-maven-plugin/maven-metadata.xml.svg?colorB=007ec6)](https://maven-badges.herokuapp.com/maven-central/com.google.cloud.tools/jib-maven-plugin)
-[![Gradle Plugin Portal](https://img.shields.io/badge/gradle%20plugin-v0.9.8-blue.svg)](https://plugins.gradle.org/plugin/com.google.cloud.tools.jib)
+[![Gradle Plugin Portal](https://img.shields.io/maven-metadata/v/http/central.maven.org/maven2/com/google/cloud/tools/jib-maven-plugin/maven-metadata.xml.svg?colorB=007ec6&label=gradle)](https://plugins.gradle.org/plugin/com.google.cloud.tools.jib)
```
Comment (coollog) [inline README.md]: extra `)` after `gradle`

Comment (loosebazooka) [reply README.md]: oh man, wrong link

Comment (loosebazooka) [reply README.md]: also not sure what you maven by extra ')'

Comment (coollog) [reply README.md]: I misread it haha

**Pull request merged**